### DISCUSSION TITLE
Abstract submission form and deadlines for the SDC User Group Workshop in September

### DISCUSSION THREAD

**Discussion by violetacln** [sdcTools/UserSupport#223] [User Group]
Please, check https://sdctools.github.io/UserSupport/  for new information about the SDC User Group _Workshop_ (September 20, 2022, at INSEE)  regarding main deadlines and abstract submission form! 
Looking forward to your contributions!

**Comment by AncoHundepool** [comment]
Dear Violetta,

 

Thank you very much for including me in this mail.

Just a question.

Last time you organised this meeting as a ZOOM meeting.

Do you intend to do this again, or maybe as a hybrid meeting?

 

Thanks for your attention.

 

Best regards,

Anco

 

 

 

Van: Violeta Calian 
Verzonden: 26 April 2022 12:32
CC: Subscribed ***@***.***>
Onderwerp: [sdcTools/user-group] Abstract submission form and deadlines for
the SDC User Group Workshop in September (#18)

 

Please, check https://sdctools.github.io/UserSupport/ for new information
about the SDC User Group Workshop (September 20, 2022, at INSEE) for new
information regarding main deadlines and abstract submission form!
Looking forward to your contributions!

-
Reply to this email directly, view it on GitHub
<https://github.com/orgs/sdcTools/teams/user-group/discussions/18> , or
unsubscribe
<https://github.com/notifications/unsubscribe-auth/AFLUNX3PP5W6ON2NJ3EDJ6TVG
7A2JANCNFSM5ULMBUOQ> .
You are receiving this because you are subscribed to this thread.
<https://github.com/notifications/beacon/AFLUNXYII4FFTOEG6BXXRILVG7A2JA5CNFS
M5ULMBUO2YY3PNVWWK3TUL52HS4DFVZCGS43DOVZXG2LPNZIG643UVJRW63LNMVXHIX3JMTHAAA5
I3Q.gif> Message ID: ***@***.***
***@***.***> >

**Comment by violetacln** [comment]
Hi!

Thank you for asking!

The recent messages I sent to the SDC Tools - team on github is about the User Workshop (new!).
This is different from the annual User Meeting which was indeed virtual until now.

We will have the Workshop in person, on the 20th of September, at INSEE, which kindly offerred to be the host, just before the PSD conference 21-23 September.
See details at https://sdctools.github.io/UserSupport/

Best wishes,
violeta



Violeta Calian
Aðferðafræðingur | Chief Methodologist
Stjórnsýsla og samstarf | Administration & cooperation

HAGSTOFA ÍSLANDS | STATISTICS ICELAND
Borgartún 21a - 105 Reykjavík
Beinn sími | Tel: +354 528 +354 528 1036

hagstofa.is<https://hagstofa.is/> | statice.is<https://statice.is/>
